src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/details_card.html.twig

```diff
@@ -0,0 +1,10 @@
+<div class="card card--details">
```

Comment:
```suggestion
<div class="card card-details">
```

Reply:
Fixed in https://github.com/PrestaShop/PrestaShop/pull/17617/commits/880b0ffdaca18f032d2cb941f37e0751b4745479